src/agent0/core/hyperdrive/interactive/hyperdrive_agent.py

```diff
@@ -979,6 +980,8 @@ def get_positions(
             Whether to show positions closed positions (i.e., positions with zero balance). Defaults to False.
             When False, will only return currently open positions. Useful for gathering currently open positions.
             When True, will also return any closed positions. Useful for calculating overall pnl of all positions.
+        calc_pnl: bool
+            Whether to return the pnl for the current position. Only used if the chain config's `calc_pnl` is False.
```

Comment:
I think it would be a good idea to add a conditional right off the bat and warn the user if they're setting this flag without setting chain config's `calc_pnl`. e.g.:

```py
if self.chain.config.calc_pnl and calc_pnl:
    logging.warning("calc_pnl parameter is set but will not be used because chain.calc_pnl is False.")
```